lib/crate_ruby/client.rb

```diff
@@ -22,98 +21,103 @@
 require 'json'
 require 'net/http'
 module CrateRuby
+  # Client to interact with Crate.io DB
   class Client
-    DEFAULT_HOST = "127.0.0.1"
-    DEFAULT_PORT = "4200"
+    DEFAULT_HOST = '127.0.0.1'.freeze
+    DEFAULT_PORT = '4200'.freeze
 
-    attr_accessor :logger
+    attr_accessor :logger, :schema
 
     # Currently only a single server is supported. Fail over will be implemented in upcoming versions
     # @param [Array] servers An Array of servers including ports [127.0.0.1:4200, 10.0.0.1:4201]
-    # @param [opts] Optional paramters
+    # @param [opts] opts Optional parameters
     # * logger: Custom Logger
     # * http_options [Hash]: Net::HTTP options (open_timeout, read_timeout)
+    # * schema [String]: Default schema to search in
     # @return [CrateRuby::Client]
     def initialize(servers = [], opts = {})
       @servers = servers
       @servers << "#{DEFAULT_HOST}:#{DEFAULT_PORT}" if servers.empty?
       @logger = opts[:logger] || CrateRuby.logger
       @http_options = opts[:http_options] || { read_timeout: 3600 }
+      @schema = opts[:schema] || 'doc'
     end
 
     def inspect
-      %Q{#<CrateRuby::Client:#{object_id}>}
+      %(#<CrateRuby::Client:#{object_id}>)
     end
 
     # Creates a table
     #     client.create_table "posts", id: [:integer, "primary key"], my_column: :string, my_integer_col: :integer
     # @param [String] table_name
     # @param [Hash] column_definition
-    # @option column_definition [String] key sets column name, value sets column type. an array passed as value can be used to set options like primary keys
+    # @option column_definition [String] key sets column name, value sets column type. an array passed as value can
+    # be used to set options like primary keys
     # @return [ResultSet]
     #
-    def create_table(table_name, column_definition = {}, blob=false)
+    def create_table(table_name, column_definition = {}, _blob = false)
       cols = column_definition.to_a.map { |a| a.join(' ') }.join(', ')
-      stmt = %Q{CREATE TABLE "#{table_name}" (#{cols})}
+      stmt = %{CREATE TABLE "#{table_name}" (#{cols})}
       execute(stmt)
     end
 
     # Creates a table for storing blobs
     # @param [String] name Table name
-    # @param [Integer] shard Shard count, defaults to 5
-    # @param [Integer] number Number of replicas, defaults to 0
+    # @param [Integer] shard_count Shard count, defaults to 5
+    # @param [Integer] replicas Number of replicas, defaults to 0
     # @return [ResultSet]
     #
     # client.create_blob_table("blob_table")
-    def create_blob_table(name, shard_count=5, replicas=0)
-      stmt = "create blob table #{name} clustered into #{shard_count} shards with (number_of_replicas=#{replicas})"
-      execute stmt
+    def create_blob_table(name, shard_count = 5, replicas = 0)
+      stmt = 'create blob table ? clustered into ? shards with (number_of_replicas=?)'
```

Comment:
cannot use parameter substitution for table identifier:
```rb
stmt = 'CREATE BLOB TABLE "#{name}" CLUSTERED INTO ? SHARDS WITH (number_of_replicas=?)'
```

also, note the double-quotes for #{name} -- otherwise if `name = 'TableName'` CrateDB would lowercase it